os/net/lwip/src/api/sockets.c

```diff
@@ -444,6 +444,9 @@ struct lwip_sock *trycheck_selwait_socket(int s, pid_t pid)
 {
 	struct lwip_sock *sock;
 	sock = tryget_socket(s, pid);
+	if (sock == NULL) {
```

Comment:
is this scenario created or tested?

Reply:
without checking this, few new TCs may stuck here.